torch_xla/csrc/tensor.h

```diff
@@ -24,6 +24,38 @@
 
 namespace torch_xla {
 
+class ExecutionContext {
```

Comment:
As discussed this should be a debug only mode. Would it make sense to check a simple `static bool debug_egaer_mode` within a function? I want to avoid introducing new Context class unless we have to.

Reply:
Yeah, if go to using an env variable, it would make sense to just use a simple static bool and avoid the api thing and context creation.